README.md

```diff
@@ -30,6 +30,24 @@ $ npm run start:hmr
 $ npm run start:prod
 ```
 
+## Database migration workflow
+Automatic migration generation creates a new migration file and writes all sql queries that must be executed to make a new database or to update the database.
+```bash
+$ node_modules/.bin/ts-node ./node_modules/.bin/typeorm migration:generate -n 'name-of-choice'
+```
+The file that will be created can be found at path chosen in ormconfig.json
+The rule of thumb is to generate a migration after each entity change.
+
+To execute all pending migrations use following command
+```bash
+$ node_modules/.bin/ts-node ./node_modules/.bin/typeorm migration:run
```

Comment:
Maybe find a way to make the command shorter in the future (but not via global install 😛), but not the highest priority right now imo.